python/src/aiconfig/editor/client/src/components/prompt/RunPromptButton.tsx

```diff
@@ -41,4 +42,16 @@ export default memo(function RunPromptButton({
       )}
     </Button>
   );
+
+  return !disabled ? (
+    button
+  ) : (
+    <Tooltip
+      label="Can't run while another prompt is running :("
+      offset={20}
+      withArrow
+    >
+      <span>{button}</span>
```

Comment:
Does the span affect the placement (vs using div)? span is inline while div is block so just want to double-check it doesn't affect anything

Reply:
It does actually affect placement, that's why I had to give toooltip offset. But yea I'll try with `<div>` instead thanks for the tip!